website/static/js/pages/base-page.js

```diff
@@ -111,6 +112,7 @@ $(function() {
         $osf.initializeResponsiveAffix();
     }
     new NavbarControl('.osf-nav-wrapper');
+    new DevModeHelper('.dev-mode-helper');
```

Comment:
Minor: Better to bind by ID rather than class (on the off-chance the the class is used more than once).